crates/guest-rust/macro/src/lib.rs

```diff
@@ -140,6 +143,16 @@ impl Parse for Config {
                     Opt::DisableCustomSectionLinkHelpers(disable) => {
                         opts.disable_custom_section_link_helpers = disable.value();
                     }
+                    Opt::Debug(enable) => {
+                        debug = enable.value();
+                    }
+                    Opt::Async(val, span) => {
+                        if async_configured {
+                            return Err(Error::new(span, "cannot specify second async config"));
+                        }
+                        async_configured = true;
+                        opts.async_ = val;
+                    }
```

Comment:
Should this test for `if cfg!(feature = "async")` and generate an error if it's not turned on?

Reply:
Good catch, I had a test in there earlier, but removed it over-zealously later.  Just pushed an update to add it back.